packages/serverpod_shared/lib/src/config.dart

```diff
@@ -386,6 +401,52 @@ class RedisConfig {
   }
 }
 
+/// Configuration for session logging.
+class SessionLogConfig {
+  /// True if persistent logging (e.g., to Redis) should be enabled.
+  final bool persistentEnabled;
+
+  /// True if console logging should be enabled.
+  final bool consoleEnabled;
+
+  /// Creates a new [SessionLogConfig].
+  SessionLogConfig({
+    required this.persistentEnabled,
+    required this.consoleEnabled,
+  });
+
+  factory SessionLogConfig._fromJson(
+    Map logsSetup,
```

Comment:
Suggestion: should we maybe just call this `sessionLogConfigJson` to be consistent with naming?

Reply:
Yes, done, thank u!